src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/CSharpTestLspServer.cs

```diff
@@ -187,16 +188,16 @@ static DidChangeTextDocumentParams CreateDidChangeTextDocumentParams(Uri documen
 
     #endregion
 
-    private class RazorCapabilitiesProvider : IRazorCapabilitiesProvider
+    private class RazorCapabilitiesProvider : IRazorTestCapabilitiesProvider
```

Comment:
`IRazorCapabilitiesProvider` was marked obsolete.

Reply:
Thanks, noticed this change merged in last night, and was going to respond today :)

Reply:
Turns out this breaks running tooling tests locall, so going to have to revert. We need to wait for these roslyn bits to be in a VS build

Reply:
Nope, even reverting this, just having the new roslyn dep breaks running unit tests for me. I don't know what to think any more.